.github/workflows/ci-dgraph-code-coverage.yml

```diff
@@ -0,0 +1,74 @@
+name: ci-dgraph-code-coverage
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+    branches:
+      - main
+  schedule:
+    - cron: "30 0 * * *"
```

Comment:
Once a day + on every PR

Reply:
Make the cron * 22 * * *  to run everyday 10pm

Reply:
That's 0 22 * * *